interface/forms/gad7/info.txt

```diff
@@ -0,0 +1 @@
+GAD-7
```

Comment:
if you add `Clinical` to the 2nd line here it will register and load the form under Clinical which may be more appropriate?

https://github.com/openemr/openemr/commit/ce9ebc54be3305d99c23b697ec1af282c61e7cd3

Reply:
thanks - we are using it under 'clinical', I hadn't realised this could be automatically registered. will do.